### PR TITLE
Small fixes [failing due to LIBSVM compat bound]

### PR DESCRIPTION
**Pull request by tlienart**
* addresses #154 (randomforest* input scitype) + a fix for LinearRegressor in GLM

### CODE REVIEW

**Comment by tlienart** [comment]
Not passing due to shitty compat bounds on LIBSVM...

File: src/GLM.jl [hunk 1]
```diff
@@ -187,7 +187,7 @@ metadata_pkg.((LinearRegressor, LinearBinaryClassifier, LinearCountRegressor),
 
 metadata_model(LinearRegressor,
     input=MLJBase.Table(MLJBase.Continuous),
-    output=AbstractVector{MLJBase.Continuous},
+    target=AbstractVector{MLJBase.Continuous},
     weights=false,
```
Comment (ablaom) [inline src/GLM.jl]: Good catch!

**Comment by ablaom** [comment]
I don't understand this at all. 

LIBSVM.jl has no Project file, only a REQUIRE file. According to the old [julia documentation](https://docs.julialang.org/en/v0.6/manual/packages/#Requirements-Specification-1) the versions specified there are just *lower* bounds. Since it specifies "Compat 0.41.0" there should be no prohibition on Compat 3.0, right? Indeed this is confirmed by previous builds of MLJModels, where testing (of LIBSVM wrapper) carried on happily with Compat 2.2. But, if in a new environment I `add` MLJModels (which puts Compat 3.0 in the manifest) then doing `add LIBSVM` forces Compat to drop from 3.0 to 2.2. Why?

**Comment by ablaom** [comment]
Just restarting the build after merging #159

**Pull request merged**